normandy/control/static/control/js/components/RecipeForm.jsx

```diff
@@ -42,6 +97,22 @@ class RecipeForm extends React.Component {
             <textarea field={filter_expression} {...filter_expression} />
           </div>
         </div>
+        <div className="row">
+          <div className="fluid-3">
```

Comment:
I think this entire `div` could be a standalone stateless component to help make the code more readable:

```jsx
<ActionNameSelect 
  className="fluid-3" 
  field={action_name}
  onChange={this.changeAction}
/>
```